requirements/dev.txt

```diff
@@ -8,8 +8,7 @@ pytest-cov==2.10.1
 # mypy adds new rules in new minor versions, which could cause our PR check to fail
 # here we fix its version and upgrade it manually in the future
 mypy==0.790
-# 1.17.91 has a bug, https://github.com/vemel/mypy_boto3_builder/pull/82
-boto3-stubs[essential]==1.17.90.post1
+boto3-stubs[apigateway,cloudformation,ecr,iam,lambda,s3,schemas,secretsmanager,signer,stepfunctions,sts,xray]
```

Comment:
Similar to other libraries, can we pin the version to the latest (1.21.21) instead?
This will make CI/builds more stable and reproducible. We can manually update the version if needed in the future without worrying about breaking changes.

Reply:
Just did. However, `boto3` version is not frozen in `requirements/base.txt`, so the latest `boto3` version is installed. Ideally `boto3` and `boto3-stubs` versions should be in sync.

Once `boto3` version reaches `1.22.x` it would make sense to bump `boto3-stubs` as well.